src/frontend/styles/components/sidebar-right.pcss

```diff
@@ -0,0 +1,16 @@
+.docs-sidebar--right {
```

Comment:
what is this? we already have fixed right column, you don't need to care about it.